terror_error.go

```diff
@@ -84,23 +93,7 @@ func (e *Error) Code() ErrCode {
 // The error code is a 3-tuple of abbreviated component name, error class and error code,
 // joined by a colon like {Component}:{ErrorClass}:{InnerErrorCode}.
 func (e *Error) RFCCode() RFCErrorCode {
-	ec := e.Class()
```

Comment:
We should return the `e.codeText` if the `e.codeText` had been assigned a value.